Azure/azure-cli
# {SQL VM} Update Python SDK to 1.0.0b3

**Related command**


**Description**

Released Python package 1.0.0b3 for SQL VM 2021-11-01-preview on 6/5/2022.
This API update renamed the following enum values: AssessmentDayOfWeek, AutoBackupDaysOfWeek.
This PR has no custom facing breaking changes, as it only updates the enum value reference for AssessmentDayOfWeek.
We have yet to implement CLI commands for AutoBackupDaysOfWeek, so this enum value is untouched in this PR.

Related links:
https://github.com/Azure/sdk-release-request/issues/2871
https://pypi.org/project/azure-mgmt-sqlvirtualmachine/1.0.0b3/



**Testing Guide**

```
> azdev test sqlvm 
```
Confirmed `Everyday` is no longer an accepted value for assessment-day-of-week
```
> az sql vm update -g <TESTRG> -n <TESTVM> --assessment-day-of-week Everyday --assessment-weekly-interval 3 –-assessment-start-time-local "23:00"
az sql vm update: 'Everyday' is not a valid value for '--assessment-day-of-week'. Allowed values: Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday.
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
